### DISCUSSION TITLE
Использовать Wireguard только для заблокированных ресурсов

### DISCUSSION THREAD

**Discussion by ArchimedRT** [shvchk/padavan-wireguard-client#5] [Вопросы и ответы] [answered]
Собственно, как это сделать?

Есть такая инструкция для OpenWrt:
https://habr.com/ru/post/440030/

Завести самому не удалось, т.к. не хватает знаний.

**Comment by shvchk** [comment, marked as answer]
Описанный конкретно в той статье подход "в лоб" уже, к сожалению, устарел и не охватит все блокируемые ресурсы: теперь блокировка осуществляется с помощью DPI по доменам, а не только IP. А домены могут резолвиться в разные IP (сделайте `nslookup google.com` дважды и оба раза вы получите разные наборы IP), поэтому проблематично создать фиксированный список IP. Теперь нужно использовать, например, подход подобный VPN Антизапрет: https://bitbucket.org/anticensority/antizapret-vpn-container (см. раздел Нестандартный способ маршрутизации), хотя и у него есть ограничения (например, клиенты должны использовать DNS роутера, а это не всегда возможно гарантировать).
